sellotape/sellotape_dj/main_app/templates/navigation.html

```diff
@@ -11,7 +11,10 @@
           </li>
           <li class="nav-item">
             <a class="nav-link" href="#">Explore <span class="badge badge-light">4</a>
+              {% if user.is_authenticated %}
           </li>
```

Comment:
There's a dislocation here: move the {% if one row downwards, otherwise unlogged users might encounter inconsistency with the navigation bar (you dont include the </li> of the previous menu item if user is not authenticated)